test/unit/sql/data-types.test.js

```diff
@@ -1604,7 +1677,6 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       describe('GEOMETRY', () => {
         testsql('GEOMETRY', DataTypes.GEOMETRY, {
           default: 'GEOMETRY',
-          snowflake: 'GEOGRAPHY',
```

Comment:
Did you find something about this in the snowflake PRs? Or did we not spot this before?

Reply:
It looked that GEOMETRY was mapped to GEOGRAPHY even though we have a GEOGRAPHY DataType, so I looked into `snowflake/data-types.js` and found no reference to GEOGRAPHY.

Turns out the test is disabled because `SnowflakeDialect.supports.GEOMETRY` is false, so I removed this broken test to avoid confusing the person that ends up enabling the GEOMETRY DataType in snowflake